styles/global.css

```
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
```

Stylelint (17.2.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap'" (import-notation)

(import-notation)